exercises/E-conditionals/exercise.js

```diff
@@ -9,8 +9,16 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole) {
```

Comment:
I think this should be `if (danielsRole === 'mentor')` as what you have put would always evaluate to true, even if the value of `danielsRole` changed